docs/api/spaces-management/copy_saved_objects.asciidoc

```diff
@@ -0,0 +1,288 @@
+[role="xpack"]
+[[spaces-api-copy-saved-objects]]
+=== Copy Saved Objects to Space
+++++
+<titleabbrev>Copy Saved Objects to Space</titleabbrev>
+++++
+
+experimental[This functionality is *experimental* and may be changed or removed completely in a future release.]
+
+Copies saved objects from one space to other spaces.
+
+////
+Use the appropriate heading levels for your book.
+Add anchors for each section.
+FYI: The section titles use attributes in case those terms change.
+////
+
+[[spaces-api-copy-saved-objects-request]]
+==== {api-request-title}
```

Comment:
Instead of `{api-request-title}`, it should be `Request` or whatever title you want it to be. For the other Kibana APIs, I just used `Request`, but I'm always open for other ideas.

Reply:
Oh interesting, I thought the purpose of using `{api-request-title}` was to ensure the section titles were consistent across all docs, with a central definition. Am I misunderstanding this comment?

> FYI: The section titles use attributes in case those terms change.

FWIW, it seems to be replacing the attributes correctly:
![image](https://user-images.githubusercontent.com/3493255/63177511-0fe52600-c016-11e9-8f9c-40f1097fdb12.png)